### ISSUE TITLE
PhantonJS tests: Unable to access network

### ISSUE THREAD

**Issue by beautifulcoder** [knockout/knockout#1765]
When I type this after cloning the repo:

```
grunt
```

I see:

```
Running "test:phantomjs" (test) task
Running Knockout tests in Phantom.js
Unable to access network
```

The README documentation says you need PhantomJS, which I have, but it does not work.

**Comment by SteveSanderson** [comment]
I'm on PhantomJS 1.9.6, and I also found that trying to upgrade to a newer version caused problems.

For now can you use Phantom 1.9.6? Or if you can work out what we need to change for the tests to run in newer versions, better still!

**Comment by brianmhunt** [comment]
Incidentally, this should be solved via the proposed gulp build system in #1360 .

**Comment by beautifulcoder** [comment]
Turns out, it was a file URI issue. Here is a pull request:

https://github.com/knockout/knockout/pull/1776

**Comment by brianmhunt** [comment]
Thanks @beautifulcoder ; this is indeed solved in #1360, but I am not sure when that'll be merged. If that is not going to be for a while we should probably merge #1776 (and there seems to be no harm to merging it).

**Comment by mbest** [comment]
Closing in favor PR #1776
